exist-core/src/main/java/org/exist/dom/QName.java

```diff
@@ -326,7 +325,7 @@ public static QName parse(final String namespaceURI, final String qname) throws
         }
     }
 
-    private final static Pattern ptnClarkNotation = Pattern.compile("\\{([^&{}]*)\\}([^&{}:]+)");
+    private final static Pattern ptnClarkNotation = Pattern.compile("\\{([^&{}]*)}([^&{}:]+)");
```

Comment:
Is this still the same pattern?

Reply:
Yes, as I understand it, without an opening curly brace, as it is still escaped, the closing one is treated as a literal.